apps/company/views.py

```
@@ -1,36 +1,74 @@
 from django.shortcuts import get_object_or_404
+from django.core.paginator import Paginator
 from drf_spectacular.types import OpenApiTypes
-from drf_spectacular.utils import extend_schema, OpenApiResponse, OpenApiParameter
+from drf_spectacular.utils import extend_schema, OpenApiResponse, OpenApiParameter, OpenApiExample
+from apps.product.tasks import *
```

Ruff

`from apps.product.tasks import *` used; unable to detect undefined names

(F403)